config/husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# config/husky/pre-commit
+# Run `lint-staged` before every commit.
+
+. "$(dirname "$0")/_/husky.sh"
+
+FORCE_COLOR=2 pnpm lint-staged
```

Comment:
What does `FORCE_COLOR` do?

Reply:
Colour libraries disable colours when `CI` is truthy, so this force-enables that.

Level 2 is 256 colour support.

Reply:
Thank you!

Reply:
It seems git (or pnpm?) disables color output when running the hooks, so this force-enables it.

This happens locally, not in CI.